tools/cli/go-whisk-cli/commands/action.go

```diff
@@ -850,6 +851,39 @@ func printActionDeleted(entityName string) {
             }))
 }
 
+// Check if the specified action is a web-action
+func IsWebAction(client *whisk.Client, qname *QualifiedName) bool {
+    var isWebAction = false;
+
+    savedNs := client.Namespace
+    client.Namespace = qname.namespace
+    fullActionName := "/" + qname.namespace + "/" + qname.entityName
+
+    action, _, err := client.Actions.Get(qname.entityName)
+    if err != nil {
+        whisk.Debug(whisk.DbgError, "client.Actions.Get(%s) error: %s\n", fullActionName, err)
+        whisk.Debug(whisk.DbgError, "Unable to obtain action '%s' for validation\n", fullActionName)
+    } else {
+        weVal := getValue(action.Annotations, "web-export")
```

Comment:
Seems like it'd be much cleaner to just do `isWebAction, _ = getValue(action.Annotations, "web-export").(bool)` and return whatever `isWebAction` is without all the conditional checking.